src/main/java/com/urswolfer/gerrit/client/rest/http/changes/ChangesRestClient.java

```diff
@@ -40,21 +40,24 @@
     private final DiffInfoParser diffInfoParser;
     private final SuggestedReviewerInfoParser suggestedReviewerInfoParser;
     private final ReviewerInfoParser reviewerInfoParser;
+    private final EditInfoParser editInfoParser;
 
     public ChangesRestClient(GerritRestClient gerritRestClient,
                              ChangesParser changesParser,
                              CommentsParser commentsParser,
                              FileInfoParser fileInfoParser,
                              DiffInfoParser diffInfoParser,
                              SuggestedReviewerInfoParser suggestedReviewerInfoParser,
-                             ReviewerInfoParser reviewerInfoParser){
+                             ReviewerInfoParser reviewerInfoParser,
+                             EditInfoParser editInfoParser){
```

Comment:
Please add a space between ) and { (I see, it was not there before already ;) ).